### PR TITLE
Milestone: Create new component and complete the desing of calculator. 

### PR DESCRIPTION
**Pull request by Arslan49sp**
## This pull request contains the following features. 

- Remove previous boilerplate
- Create a `components` repository inside `src`
- Create a `Calculator.js` component and complete the design of the calculator. 
- Use at least one child component of the `Calculator.js`
- Use `Calculator.js` in the `App.js` component.

### CODE REVIEW

File: src/components/Calculator.js
Comment (skyv26) [inline src/components/Calculator.js:1]: - [ ] Please 🙏🏻 adhere to the project requirement, according to which your design should match with provided `wireframe`. No doubt you did an awesome job, but following the requirements at the detail level is very important. Kindly fix the colours, and make your design look big and centre on the screen.

### Requirement
image

### Design Comparison
<table>
<tr>
<th>Wireframe Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

Screenshot from 2023-09-30 06-45-41

</td>
<td>

Screenshot from 2023-09-30 06-45-23

</td>
</tr>
</table>

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+function Calculator() {
+  return (
+    <div className="calc-container">
+      <div className="result-screen">0</div>
+      {number.map((n) => {
+        if (n === '\u00F7' || n === '\u00D7' || n === '-' || n === '+' || n === '=') {
+          return <InputButton inputSymbol={n} key={n} background="red" />;
+        }
+        return <InputButton inputSymbol={n} key={n} background="gray" />;
```
Comment (skyv26) [inline src/components/Calculator.js:34]: - [ ] Kindly make sure that your application is running without any kind of warning or errors, I noticed that your project have some issue that needs to be addressed before moving further. Please take a look at the below errors and try to solve it, debugging your code will help you to be a more better developer.

image

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @Arslan49sp , 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Meaningful **Github** commits ✅ 

## Required Changes ♻️

- [ ] As per the requirement. you should delete all the boilerplate code, but you still have the `App.test.js`, `webVitals`, and `setupTest` files, in your project that need to be deleted, which comes pre-defined with the CRA template. Later on, you will learn how to add them all back with a purpose.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @Arslan49sp👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the code reviewer✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — COMMENTED** [review]
## STATUS: APPROVED :green_circle:  

Hi @Arslan49sp    :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- File structure as recommended ✅ 
- Nicely implemented functionalities ✅ 
- Good use of Gitflow ✅ 
- No linter issues :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**